samples/HttpRequestFeaturesv2/ClientApp/components/home-page/home-page.html

```diff
@@ -0,0 +1,48 @@
+<p> </p>
```

Comment:
Would it be a good idea to prevent the user from sending a rank request a second time? ie., after they send the rank request once, we disable the "send request" button until they've generated a new request?